README.md

```diff
@@ -173,17 +173,21 @@ print(response.choices[0].message)
 | Force Cache Refresh | `cache_force_refresh`   | `True`, `False` (Boolean)                                 | ❔ Optional |
 | Cache Age           | `cache_age`             | `integer` (in seconds)                           | ❔ Optional |
 | Trace ID            | `trace_id`              | `string`                                         | ❔ Optional |
-| Retries         | `retry`           | `integer` [0,5]                                  | ❔ Optional |
+| Retries         | `retry`           | `json object` - See details below                        | ❔ Optional |
```

Comment:
It'll help if we can add a JSON example here directly. Let's not worry about the table structure as it's more important I think to have these details in a single place

So, the text here can say:

`{dict}` with two required keys: **`attempts`** which expects **integers in `[0,5]`** and **`on_status_codes`** which expects array of status codes like **`[429,502]`**

Example: { "attempts": 5, "on_status_codes":[429,500] }